Formula/sonarqube.rb

```diff
@@ -1,8 +1,8 @@
 class Sonarqube < Formula
   desc "Manage code quality"
   homepage "http://www.sonarqube.org/"
```

Comment:
Audit says: `The URL http://www.sonarqube.org/ could use HTTPS rather than HTTP`